packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -0,0 +1,267 @@
+const { deepEqual } = require('assert');
+const { normalizeRoutes } = require('../');
+const {
+  convertCleanUrls,
+  convertRedirects,
+  convertRewrites,
+  convertHeaders,
+  convertTrailingSlash,
+} = require('../dist/superstatic');
+
+function routesToRegExps(routeArray) {
+  const { routes, error } = normalizeRoutes(routeArray);
+  if (error) {
+    throw error;
+  }
+  return routes.map(r => new RegExp(r.src));
+}
+
+function assertMatches(actual, matches, isExpectingMatch) {
+  routesToRegExps(actual).forEach((r, i) => {
+    matches[i].forEach(text => {
+      deepEqual(r.test(text), isExpectingMatch, `${text} ${r.source}`);
+    });
+  });
+}
+
+function assertRegexMatches(actual, mustMatch, mustNotMatch) {
+  assertMatches(actual, mustMatch, true);
+  assertMatches(actual, mustNotMatch, false);
+}
+
+test('convertCleanUrls', () => {
+  const actual = convertCleanUrls([
+    'file.txt',
+    'path/to/file.txt',
+    'file.js',
+    'path/to/file.js',
+    'file.html',
+    'path/to/file.html',
+  ]);
+  const expected = [
+    {
+      src: 'file.html',
+      headers: { Location: 'file' },
+      status: 307,
+      continue: true,
+    },
+    {
+      src: 'path/to/file.html',
+      headers: { Location: 'path/to/file' },
+      status: 307,
+      continue: true,
+    },
+    { src: 'file', dest: 'file.html', continue: true },
+    { src: 'path/to/file', dest: 'path/to/file.html', continue: true },
+  ];
+  deepEqual(actual, expected);
+
+  const mustMatch = [
```

Comment:
You should test with a leading `/`

Reply:
I think that the input from `now-deployments-loop` is going to be relative file paths which don't start with a `/`. Should all the file paths be transformed absolute before converting to clean urls??

Reply:
I can't imagine relative URLs being useful here...

Reply:
It's not a matter of being useful or not. It's the reality of how the build `outputs` are stored in the database. They are stored as relative paths.

![image](https://user-images.githubusercontent.com/229881/66839462-71e9db00-ef34-11e9-9165-9f4e206499d9.png)

Reply:
Relative to what?

Reply:
Probably relative to now.json or to the root of the repo which are currently the same. The builder chooses the output path so it could be anything.

I guess the solution here is to add any missing `/` prefix before creating the routes.

Reply:
Done